docs/src/main/asciidoc/mp/server.adoc

```diff
@@ -333,24 +333,41 @@ io.helidon.examples.AdminService:
 .META-INF/microprofile-config.properties - File system static content
 ----
 # Location of content on file system
-server.static.path.location=/var/www/html
-# default is index.html
-server.static.path.welcome=resource.html
-# static content path - default is "/"
-# server.static.path.context=/static-file
+server.features.static-content.path.0.location=/var/www/html
+# default is index.html (only in Helidon MicroProfile)
+server.features.static-content.path.0.welcome=resource.html
+# static content context on webserver - default is "/"
+# server.features.static-content.path.0.context=/static-file
 ----
 
 [source,properties]
 .META-INF/microprofile-config.properties - Classpath static content
 ----
 # src/main/resources/WEB in your source tree
-server.static.classpath.location=/WEB
+server.features.static-content.classpath.0.location=/WEB
 # default is index.html
-server.static.classpath.welcome=resource.html
+server.features.static-content.classpath.0.welcome=resource.html
 # static content path - default is "/"
-# server.static.classpath.context=/static-cp
+# server.features.static-content.classpath.0.context=/static-cp
+----
+
+It is usually easier to configure list based options using `application.yaml` instead, such as:
```

Comment:
list-based?

Reply:
such as:
```yaml
server.features.static-content.classpath:
 - context:  "/static"
   location: "/web"
 - context: "/js"
   location: "/js"
```
i.e. a list of classpath configuration for static content. In properties you must use index, such as:
```properties
server.features.static-content.classpath.0.context=/static
server.features.static-content.classpath.0.location=/web
server.features.static-content.classpath.1.context=/js
server.features.static-content.classpath.1.location=/js
```

Reply:
No, I meant that "list-based" reads better in the English sentence